src/main/java/com/epam/petclinic/clinic/controller/AnimalController.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.petclinic.clinic.controller;
+
+import com.epam.petclinic.clinic.model.Animal;
+import com.epam.petclinic.clinic.repository.AnimalRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Animal controller.
+ * <p/>
+ * Copyright (C) 2016 copyright.com
+ * <p/>
+ * Date: 4/18/2017
+ *
+ * @author Stanislau Halauniou
+ */
+@RestController
+@RequestMapping("/api/animals")
```

Comment:
No matter, done according to the https://docs.google.com/document/d/1YFDeqRO4UQ7oDVkt96fEH3Tm2nPVcH8miBbgo9zxga8/edit#